code/nodes/trainTestSplit/funcs.q

```diff
@@ -0,0 +1,45 @@
+\d .automl
+
+// Definitions of the main callable functions used in the application of .automl.trainTestSplit
+
+
+// Configuration update
+
+// @kind function
+// @category trainTestSplit
+// @fileoverview Apply TTS function
+// @param cfg      {dict}          Location and method by which to retrieve the data
+// @param feat     {tab}           The feature data as a table 
+// @param tgt      {(num[];sym[])} Numerical or symbol vector containing the target dataset
+// @param sigFeats {sym[]}         Significant features
+// @return         {dict}          Data separated into training and testing sets
+trainTestSplit.applyTTS:{[cfg;feats;tgt;sigFeats]
+  data:flip feats sigFeats;
+  ttsFunc:util.qpyFuncSearch cfg`tts;
+  testSize:trainTestSplit.sizeCheck[cfg`sz];
```

Comment:
The `sizeCheck` logic could be added earlier in the workflow, most likely in the `dataCheck` node to catch it as early as possible.

Given that the python example provided also handles integer inputs for test size do we want to limit to only floating point inputs for 3rd param?